### ISSUE TITLE
Live tiles for the gallery is not working. And the scrolling in the app list is lagging 

### ISSUE THREAD

**Issue by Mr-Jarvis3000** [groovelauncher/GrooveLauncher#20]

**Comment by berkaytumal** [comment]
what is the package name for your gallery. and what do you mean by not working does it just show an empty frame?

**Comment by Mr-Jarvis3000** [comment]
Yeah it's showing an empty frame. I can't find the package name but it's Vivo's default gallery app.

**Comment by KParthSingh** [comment]
> Yeah it's showing an empty frame.

Does this issue still persist in 0.5.2-beta.5? I was facing this issue in 0.5.0-beta.5 then updated the launcher and it works.

**Comment by Mr-Jarvis3000** [comment]
I just updated the app and the issue is still there. For just a second it tries to show pictures and then goes blank again.

**Comment by Mr-Jarvis3000** [comment]
Still getting an empty frame. No live tile for the gallery. 
Package name (com.vivo.gallery)

**Comment by berkaytumal** [comment]
![CleanShot 2025-01-03 at 00 43 21](https://github.com/user-attachments/assets/c78b49c9-92fa-45fd-aa25-4b1376dc2e16)
the package name is already in the source code. can you send a screen recording of whats happening?

**Comment by Mr-Jarvis3000** [comment]
I just reset the launcher and restart it again and now it's working 😋. But the scroll through the launcher is still very sluggish.

**Comment by KParthSingh** [comment]
Dude.
